src/planning.py

```diff
@@ -423,6 +424,65 @@ def _run_low_level_search(task: Task, option_model: _OptionModelBase,
     return [], True
 
 
+def _run_plan_with_option_model(
```

Comment:
add comprehensive tests for this function into `test_planning.py`, and remove the `pragma: no cover`

Reply:
Will do